mirdata/datasets/billboard.py

```diff
@@ -0,0 +1,480 @@
+        return self._track_metadata.get("peak_rank")
+
+    @property
+    def weeks_on_chart(self):
+        return self._track_metadata.get("weeks_on_chart")
+
+    @core.cached_property
+    def chords_full(self):
+        return load_chords(self.lab_full_path)
+
+    @core.cached_property
+    def chords_majmin7(self):
+        return load_chords(self.lab_majmin7_path)
+
+    @core.cached_property
+    def chords_majmin7inv(self):
+        return load_chords(self.lab_majmin7inv_path)
+
+    @core.cached_property
+    def chords_majmin(self):
+        return load_chords(self.lab_majmin_path)
+
+    @core.cached_property
+    def chords_majmininv(self):
+        return load_chords(self.lab_majmininv_path)
+
+    @core.cached_property
+    def chroma(self):
+        """
+        NNLS chroma vectors from the NNLS chroma Vamp plug-in
+        """
+        # removed the first column since it contains metadata.
+        with open(self.bothchroma_path, "r") as f:
+            return np.array([l for l in csv.reader(f)])[:, 1:].astype(np.float32)
+
+    @core.cached_property
+    def tuning(self):
+        """
+        Tuning from NNLS chroma Vamp plug-in
+        """
+        with open(self.tuning_path, "r") as f:
+            return next(csv.reader(f))[1:]
+
+    @core.cached_property
+    def sections(self):
+        return load_sections(
+            os.path.join(self._data_home, self._track_paths["salami"][0]),
+            section_type="letter",
+        )
+
+    @core.cached_property
+    def named_sections(self):
+        return load_sections(
+            os.path.join(self._data_home, self._track_paths["salami"][0]),
+            section_type="name",
+        )
+
+    @core.cached_property
+    def salami_metadata(self):
+        return parse_salami_metadata(
+            os.path.join(self._data_home, self._track_paths["salami"][0])
+        )
+
+    @property
+    def audio(self) -> Optional[Tuple[np.ndarray, float]]:
+        """The track's audio
+
+        Returns:
+            * np.ndarray - audio signal
+            * float - sample rate
+
+        """
+        return load_audio(self.audio_path)
+
+    def to_jams(self):
+        """Get the track's data in jams format
+
+        Returns:
+            jams.JAMS: the track's data in jams format
+
+        """
+        return jams_utils.jams_converter(
+            audio_path=self.audio_path, metadata=self._track_metadata
+        )
+
+
+@io.coerce_to_bytes_io
+def load_audio(fhandle: BinaryIO) -> Tuple[np.ndarray, float]:
+    """Load a TinySOL audio file.
+
+    Args:
+        fhandle (str or file-like): File-like object or path to audio file
+
+    Returns:
+        * np.ndarray - the mono audio signal
+        * float - The sample rate of the audio file
+
+    """
+    return librosa.load(fhandle, sr=None, mono=True)
+
+
+@io.coerce_to_string_io
+def load_chords(fhandle: TextIO):
+    """Private function to load LAB format chord data from a file
+    Args:
+        chords_path (str):
```

Comment:
Please complete here, and the docs of this function are not rendering properly

Reply:
Fixed in https://github.com/Spijkervet/mirdata/commit/04dfb46f6cc301e36b55452c16643fccdd73decf